pkg/security/probe/model.go

```diff
@@ -286,6 +286,75 @@ func (ev *Event) ResolveExecArgs(e *model.ExecEvent) string {
 	return ev.Exec.Args
 }
 
+// ResolveExecArgv resolves the args of the event as an array
+func (ev *Event) ResolveExecArgv(e *model.ExecEvent) []string {
+	if len(ev.Exec.Argv) == 0 && len(ev.ProcessContext.ArgsArray) > 0 {
+		ev.Exec.Argv = ev.ProcessContext.ArgsArray
+	}
+	return ev.Exec.Argv
+}
+
+func isAlphaNumeric(r rune) bool {
+	return (r >= 'a' && r <= 'z') || (r >= 'A' && r <= 'Z') || (r >= '0' && r <= '9')
+}
+
+// ResolveExecArgsFlags resolves the arguments flags of the event
+func (ev *Event) ResolveExecArgsFlags(e *model.ExecEvent) (flags []string) {
+	for _, arg := range ev.ProcessContext.ArgsArray {
```

Comment:
this should/will be `ev.Exec.ArgsArray`, `ProcessContext` will be the process which triggered the exec